COMPILING

```diff
@@ -183,6 +183,40 @@ can be used for building with MSBuild.  Note that you still need to run
 flex/bison using "make generated_files" before opening the project.
 
 
+WORKING WITH CMAKE (EXPERIMENTAL)
+---------------------------------
+
+There is an experimental build based on CMake instead of hand-written
+makefiles. You may prefer to use this, as it should work on a wider variety of
+systems than the standard makefile build, and can integrate better with IDEs
+and static-analysis tools.
+
+0) Run `cmake --version`. If you get a command-not-found error, or the installed
+   version is lower than 3.2, go and install a new version. Most Linux distros
+   have a package for CMake, and Mac users can get it through homebrew.
+   Windows users should download it manually from cmake.org.
+
+1) Navigate to the CBMC directory in a terminal.
```

Comment:
Get the instructions in line with the above instructions: precise commands, no fuffing about a "CBMC directory" (no idea what that is).